fabtests/multinode/include/pattern.h

```diff
@@ -41,10 +41,13 @@
 /* Initial value for iterator position. */
 #define PATTERN_NO_CURRENT (-1)
 
+/* Number of patterns to test */
+#define NUM_TESTS 4
```

Comment:
This should be tied to the size of the array, rather than defined and maintained separately.